src/browser/modules/D3Visualization/components/Inspector.jsx

```diff
@@ -39,12 +39,12 @@ import {
 import { GrassEditor } from './GrassEditor'
 import { RowExpandToggleComponent } from './RowExpandToggle'
 import ClickableUrls from '../../../components/clickable-urls'
+import thousandsCommaSeparatedNumber from '../../../../shared/utils/thousands-separated-number'
```

Comment:
I'm struggling to name this so any suggestions very welcome

Reply:
`numberToUsLocale` maybe?

Reply:
yeah thats more descriptive of what its actually doing.